src/App.js

```diff
@@ -1,24 +1,43 @@
-import logo from './logo.svg';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { Toaster } from 'react-hot-toast';
+import { currentUser } from './redux/auth/currentUserSlice';
 import './App.css';
+import Login from './components/auth/Login';
+import SignUp from './components/auth/SignUp';
+import Home from './components/home/Home';
+import CarsIndex from './components/cars/CarIndex/CarIndex';
+import CarShow from './components/cars/CarShow/CarShow';
+import Layout from './components/layout/Layout';
+import AddNewCar from './components/cars/AddNewCar/AddNewCar';
+import AddReservation from './components/reservation/AddReservation';
+import ReservationList from './components/reservation/ReservationList';
+import CarDelete from './components/cars/CarDelete/CarDelete';
 
 function App() {
```

Comment:
- [ ] While I was able to run the frontend project, the same wasn't true for the backend. This means I couldn't assert the frontend's functionality.  Kindly view the review comments that were left in the [backend ](https://github.com/muneeb3123/Fast-Lane-Racers-back-end/pull/24#pullrequestreview-1635415551)review.

The project's functionality will be taken a look at in the next review. 👍